src/Elastic.CommonSchema.Serilog/Http/HttpContextAccessorAdapter.cs

```diff
@@ -41,7 +41,7 @@ public UserAgent UserAgent
 
 				return new UserAgent
 				{
-					Name = userAgent,
+					Name = clientInfo.UA.Family,
```

Comment:
`clientInfo.UA` never null?

Reply:
Honestly I was guided by usages of `clientInfo.UA` below where version is assigned. I checked now -- this approach appears to be correct as UAParser always fills those properties out (`UA`, `OS`, `Device`): https://github.com/ua-parser/uap-csharp/blob/eca6f57a2682be60984860e0675faabc6ef60d8d/UAParser/UAParser.cs#L294